app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala

```diff
@@ -9,14 +9,26 @@ import play.api.Play.current
 
 class FacebookInstantArticlesIsPublishingTest(context: Context) extends SanityTestBase(context) {
 
-  "Facebook instant articles" should "have published something in the last 30 minutes" in {
+  "Facebook instant articles" should "have published something in the last 30 minutes (60 minutes on weekends or between 00:00 - 08:30)" in {
     val httpRequest = WS.url(s"${Config.facebookInstantArticlesHost}healthcheck/publication").withRequestTimeout(10000.millis).get()
     whenReady(httpRequest) { result =>
       assume(result.status == 200, "Cannot access publication healthcheck endpoint for FB IA")
       val json = Json.parse(result.body)
-      (json \ "lastPublished").toOption.map{ lastPublished =>
+      (json \ "lastPublished").toOption.foreach { lastPublished =>
         val lastPublishedDateTime = new DateTime(lastPublished.as[String])
-        Minutes.minutesBetween(lastPublishedDateTime, new DateTime()).isLessThan(Minutes.minutes(30)) should be(true)
+
+        val now = DateTime.now()
+        val twelveAM = new DateTime().withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0)
```

Comment:
Race condition here if the test runs at exactly midnight. e.g.

`now` = Monday 23:59:59
`twelveAM` = Tuesday 00:00:00
`eightThirtyAM` = Tuesday 08:30:00

This causes your test (`now.isBefore(eightThirtyAM) && now.isAfter(twelveAM)`) to do something other than what you expect.

Doesn't really matter in this case, but worth being aware of as a cause of nasty bugs. Simple fix: only get the current time once and then reuse your `now`, rather than calling `new DateTime()` again.